src/libs/OptionsListUtils.ts

```diff
@@ -2312,7 +2312,7 @@ function getPersonalDetailSearchTerms(item: Partial<OptionData>) {
 }
 
 function getCurrentUserSearchTerms(item: OptionData) {
-    return [item.text ?? '', item.login ?? '', item.login?.replace(CONST.EMAIL_SEARCH_REGEX, '') ?? '', 'you', 'me'];
+    return [item.text ?? '', item.login ?? '', item.login?.replace(CONST.EMAIL_SEARCH_REGEX, '') ?? '', translateLocal('common.you').toLowerCase(), translateLocal('common.me')];
```

Comment:
Why do we need `toLowerCase` here? I think the search terms are case insensitive